dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/bnd.bnd

```diff
@@ -34,4 +34,9 @@ tested.features:\
 	io.openliberty.org.eclipse.microprofile.rest.client.3.0;version=latest,\
 	io.openliberty.jakarta.annotation.2.1;version=latest,\
 	io.openliberty.jakarta.concurrency.3.0;version=latest,\
-	io.openliberty.mpTelemetry.1.0.thirdparty;version=latest
+	io.openliberty.mpTelemetry.1.0.thirdparty;version=latest,\
+        io.opentelemetry:opentelemetry-opentracing-shim;version=1.19.0.alpha,\
+        io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations;version=1.19.0.alpha,\
+        io.opentracing:opentracing-api;version=0.33.0
```

Comment:
I do not think you need these two because the relevant classes should also be in `io.openliberty.mpTelemetry.1.0.thirdparty`

Reply:
I removed opentelemetry-instrumentation-annotations, but removing opentracing-api gave local compile errors.